amd64/README.md

```diff
@@ -0,0 +1,8 @@
+# HOW TO:
+
+Since the `build.sh` file is being shared across all amd64 Dockerfiles,
+to build the docker container:
+
+```shell
+docker build -f amd64/Dockerfile .
```

Comment:
Since it's a single shell command you can write it with single back ticks in one line, like you did with the `build.sh` name above